### PR TITLE
Fix 'babel-eslint' dependency

### PR DESCRIPTION
**Pull request by mrtnzlml**
We are using caret operator everywhere else and this is a mistake.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   },
   "devDependencies": {
     "babel-cli": "^6.26.0",
-    "babel-eslint": "~8.2.2",
+    "babel-eslint": "^8.2.2",
     "babel-jest": "22.4.3",
     "babel-plugin-idx": "^2.2.0",
```
Comment (tbergq) [inline package.json]: what about this?

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
   },
   "devDependencies": {
     "babel-cli": "^6.26.0",
-    "babel-eslint": "~8.2.2",
+    "babel-eslint": "^8.2.2",
     "babel-jest": "22.4.3",
```
Comment (tbergq) [inline package.json]: Should this also be `^`?

File: package.json [hunk 3]
```diff
@@ -26,7 +26,7 @@
...
     "babel-plugin-idx": "^2.2.0",
     "babel-preset-react-native": "4.0.0",
```
Comment (tbergq) [inline package.json]: and this?

**Comment by mrtnzlml** [comment]
@tbergq WTF happened with our dependencies? 😄 Fixed hopefully everywhere...

**Comment by tbergq** [comment]
> WTF happened with our dependencies

👻 commiter 🤔

**Pull request merged**